content/en/news/releases/1.14.x/announcing-1.14/upgrade-notes/index.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Istio 1.14 Upgrade Notes
+description: Important changes to consider when upgrading to Istio 1.14.0.
+publishdate: 2022-05-11
+weight: 20
+---
+
+When you upgrade from Istio 1.13.x to Istio 1.14.0, you need to consider the changes on this page.
+These notes detail the changes which purposefully break backwards compatibility with Istio 1.14.0.
+The notes also mention changes which preserve backwards compatibility while introducing new behavior.
+Changes are only included if the new behavior would be unexpected to a user of Istio 1.13.x.
```

Comment:
I'm wondering if we want to include 1.12 in this since we now support skip level upgrades.

Reply:
Reworded into "Istio 1.13.x or Istio 1.12.x".